### ISSUE TITLE
Fetch_old has conflicting processor

### ISSUE THREAD

**Issue by jazzace** [autopkg/scriptingosx-recipes#50]
TL;DR It is safe to retire Fetch_old.pkg, through deprecation or removal.

During recent testing, I determined that the custom processor used for Fetch was out of sync with the custom processor in the parent recipe. Because of this, unpredictable behaviour was occurring at run time (it wouldn't always grab the same version of the processor). This recipe appears to predate shared custom processors, hence the reason to duplicate the processor; in a more contemporary setting, we would use the identifier/SharedProcessor reference and not duplicate the processor. That would be a potential way to "solve" this problem.

Having said this, it appears that you are now using the jleggat recipes for Fetch as a basis for your child recipes. You may have been retaining your own recipe because my Fetch.ds recipe had been using it as a parent. I have now also changed my parent to the jleggat recipe, so there are no recipes that I can find that use it as a parent. It _should_ be safe to delete it, but one could also choose to adopt a deprecation warning or turn it into a stub recipe that points to the jleggat equivalent. Regardless, the custom processor that exists belong side of that recipe can and should be removed.
